### ISSUE TITLE
Uncaught TypeError: Object #<Object> has no method 'twit' 

### ISSUE THREAD

**Issue by eighteight** [StanScates/Tweet.js-Mod#56]
I seem to have followed the directions, but get the "Uncaught TypeError: Object #<Object> has no method 'twit' ". Please look at the http://post.scriptum.ru to see this error. Thank you very much.

**Comment by wtadesign** [comment]
Your ".repertory" class in your JS does not seem to match up to the div id in the html page which has 
<div id="scriptumBlock">

They  should match up. Try that first.

**Comment by eighteight** [comment]
Hi! There is&lt;div class=&quot;repertory&quot;&gt;&lt;/div&gt; element and it is being referred to as $(".repertory"). Sorry what is not matching? 

Thanks.

**Comment by wtadesign** [comment]
hi

This JS script code in your head section  the class .repertory, there is no div associated with it. 
Try changing it to #scriptumBlock which you have used for the twitter feed.

try this

$(document).ready(function() {
                $("#scriptumBlock").twit({
                    userName: "postscriptums",
                    modpath: "/js/jquery/jquerytw/",
                    numTweets: 5,
                    loaderText: "Загрузка репертуара...",
                    slideIn: true,
                    showHeading: false,
                    headingText: "",
                    showProfileLink: false,
                    slideIn: true,
                    showHeading: false,
                    headingText: "",
                    showProfileLink: false
                });
            });

**Comment by eighteight** [comment]
Same error. Have a look.

**Comment by wtadesign** [comment]
ok 

try this it a ticker version, this work for me, so as a test drop this in your code.

<script type='text/javascript'>
jQuery(function($){
$("#scriptumBlock").tweet({
username: "postscriptums",
page: 1,
avatar_size: 22,
count: 20,
loading_text: "loading ..."
})
});
$(document).ready(function() {
var ul = $(this).find(".tweet_list");
var ticker = function() {
setTimeout(function() {
var top = ul.position().top;
var h = ul.height();
var incr = (h / ul.children().length);
var newTop = top - incr;
if (h + newTop <= 0) newTop = 0;
ul.animate( {top: newTop}, 500 );
ticker();
}, 7000);
};
ticker();
$('#scriptumBlock a').attr('target','_blank');
    
});
</script>

**Comment by eighteight** [comment]
Hi, I think the part of your message got eaten. Would you like to try https://gist.github.com/

?

Thanks.

**Comment by wtadesign** [comment]
no probs

https://gist.github.com/wtadesign/1c3f0fdec1f20cae6269

**Comment by wtadesign** [comment]
only other thing is the the structure of the folder setup for the twitter folder

//
1. Acquire /twitter/. How you wish to structure the files is up to you, but by default it is designed to be placed in the docroot of your website as a whole. (IE, domain.com/twitter).
//

currently your code is

<script src="/js/jquery/jquerytw/jquery.tweet.js"></script>

it might be helpful to have as this - as default set up in the index php twitter folder is as

<script src="/twitter/jquery.tweet.js"></script>

**Comment by eighteight** [comment]
Right, moved past that error. Now I am getting "Uncaught TypeError: Cannot read property 'statuses' of null", which, perhaps is related to authentication.

Thanks a lot!

**Comment by wtadesign** [comment]
No problem, just check that your index.php (in the twitter folder) app consumer and access keys/ tokens are have been setup and matchup.

**Comment by eighteight** [comment]
For the record: the last error was due to old jquery. Upgrading to current version fixed it.

Thanks for your help.

**Comment by wtadesign** [comment]
Glad you fixed it, always a pain to setup the first time. cheers
